modules/drivers/numpy/wscript

```diff
@@ -34,10 +34,9 @@ def configure(conf):
                        includes=inc_dir,
                        libpath=lib_dir,
                        msg='Checking for library numpy' + warningmessage, 
-                       mandatory=False) 
-            
+                       mandatory=True)
```

Comment:
Fixed, it just had to bring in PYEXT when numpyconfig.h didn't. I still think mandatory needs to be true to stop PY_HAVE_NUMPY from getting defined when it's not supposed to, though.